containers/allan_dip_bot/run_bot.py

```diff
@@ -0,0 +1,192 @@
+__author__ = "Kartik Shenoy"
+__email__ = "kartik.shenoyy@gmail.com"
+
+import sys, os
+
+sys.path.append("..") # Adds higher directory to python modules path.
```

Comment:
What is the purpose of this code? If it is for testing, it should go in tests/

Reply:
It is not for testing. This is the entrypoint to the bot when using the Docker container. Please check: https://github.com/SHADE-AI/diplomacy-playground/tree/dev/bots/dipnet_sl